Resources/Prototypes/Reagents/chemicals.yml

```diff
@@ -13,6 +13,254 @@
+  physicalDesc: strong-smelling
+  color: "#a1000b"
+  boilingPoint: 78.2 # This isn't a real chemical...
+  meltingPoint: -19.4
+  plantMetabolism:
+    - !type:AdjustToxins
+      amount: 20
+    - !type:AdjustWeeds
+      amount: -4
+    - !type:AdjustHealth
+      amount: -8
+
+- type: reagent
+  id: chem.TableSalt
+  name: table salt
+  desc: Commonly known as salt, Sodium Chloride is often used to season food or kill borers instantly.
+  physicalDesc: grainy
+  color: "#a1000b"
+  boilingPoint: 1465.0
+  meltingPoint: 800.7
+  plantMetabolism:
+    - !type:AdjustWater
+      amount: -3
+    - !type:AdjustNutrition
+      amount: -0.3
+    - !type:AdjustToxins
+      amount: 8
+    - !type:AdjustWeeds
+      amount: -2
+    - !type:AdjustPests
+      amount: -1
+    - !type:AdjustHealth
+      amount: -2
+
+- type: reagent
+  id: chem.Thermite
+  name: thermite
+  desc: A mixture that becomes extremely hot when ignited, and which can burn straight through walls when applied and ignited. It'll slowly inflict burn damage to anybody dumb enough to ingest it, but can't be ignited inside inside said dumb person.
+  physicalDesc: grainy
+  color: "#757245"
+  boilingPoint: 2977.0 # Aluminum oxide
+  meltingPoint: 2030.0
+  tileReactions:
+    - !type:FlammableTileReaction
+      temperatureMultiplier: 1.35
+
+- type: reagent
+  id: chem.Toxin
+  name: toxin
+  desc: A Toxic chemical.
+  color: "#cf3600"
+  physicalDesc: opaque
+  plantMetabolism:
+    - !type:AdjustToxins
+      amount: 10
+
+- type: reagent
+  id: chem.Sugar
+  name: sugar
+  desc: Sweet.
+  color: "#ffffff"
+  physicalDesc: opaque
+  plantMetabolism:
+    - !type:AdjustNutrition
+      amount: 0.1
+    - !type:AdjustWeeds
+      amount: 2
+    - !type:AdjustPests
+      amount: 2
+
+- type: reagent
+  id: chem.SulfuricAcid
+  name: sulfuric acid
+  desc: A highly corrosive, oily, colorless liquid.
+  physicalDesc: oily
+  color: "#BF8C00"
+  boilingPoint: 337.0
+  meltingPoint: 10.31
+  plantMetabolism:
+    - !type:AdjustToxins
+      amount: 10
+    - !type:AdjustWeeds
+      amount: -2
+    - !type:AdjustHealth
+      amount: -4
+
+- type: reagent
+  id: chem.UnstableMutagen
+  name: unstable mutagen
+  desc: Causes mutations when injected into living people or plants. High doses may be lethal, especially in humans.
+  physicalDesc: glowing
+  color: "#00ff5f"
+  boilingPoint: 340282300000000000000000000000000000000 # Ethidium bromide, which doesn't boil.
+  meltingPoint: 261.0
+- type: reagent
+  id: chem.Ammonia
+  name: ammonia
+  desc: An effective fertilizer which is better than what is available to the botanist initially, though it isn't as powerful as Diethylamine
+  physicalDesc: pungent
+  color: "#77b58e"
+  boilingPoint: -33.0
+  meltingPoint: -77.7
+  plantMetabolism:
+    - !type:AdjustNutrition
+      amount: 1
+    - !type:AdjustHealth
+      amount: 0.5
+
+- type: reagent
+<<<<<<< HEAD
```

Comment:
Just assume that any of these kinds of lines is a merge conflict.

Reply:
they are